### DISCUSSION TITLE
What's the difference between full_addr and victim_addr in update_replacement_state()?

### DISCUSSION THREAD

**Discussion by saffyhala** [ChampSim/ChampSim#531] [Q&A] [answered]
Could anyone please clarify what is the difference in the parameters full_addr and victim_addr in the update_replacement_state() function? 
_Is full_addr the address that replaces the evicted address victim_addr?_

**Comment by maccoymerrell** [comment, marked as answer]
There is some documentation here:
https://champsim.github.io/ChampSim/develop/Modules.html#_CPPv424update_replacement_state8uint32_tllN8champsim7addressEN8champsim7addressE11access_typeb

*Note: This is documentation for our develop branch, but the parameters should not be too different from master.
